### ISSUE TITLE
Mix between left potentiometer and potentiometer number 3

### ISSUE THREAD

**Issue by Franjose1984** [FrSkyRC/ETHOS-Feedback-Community#980]
Hi, I just realized that there is a little mix between the left pot and the 3 pot, I have the latest version installed (1.0.11 EU) in x20s

**Comment by loetefix** [comment]
I can not confirm. I tested with 1.0.11EU on X20.
No mix between the pots.

**Comment by Franjose1984** [comment]
Here I put some pictures of the problem, the left pot is channel 13 and pot 3 is channel 15. I have checked all the mixes in the transmitter and everything is correct, there is no mix between these two channels
![20210921_151713](https://user-images.githubusercontent.com/78416937/134188762-2156520e-
![20210921_151724](https://user-images.githubusercontent.com/78416937/134188800-b28c8d7c-78e7-4343-a35b-7d33b3ba5ec3.jpg)
16c2-417a-894b-4d060f5a9e4b.jpg)
![20210921_151730](https://user-images.githubusercontent.com/78416937/134188820-dd38f80e-87af-4267-907a-78cc31c2b0b2.jpg)
.

**Comment by Franjose1984** [comment]
yes and the same thing keeps happening, and i have calibrate again.
![20210921_161027](https://user-images.githubusercontent.com/78416937/134198173-d44cce81-5735-449e-accb-37625e2daf70.jpg)
![20210921_161037](https://user-images.githubusercontent.com/78416937/134198178-960cb7e7-eb67-466b-a63e-f3268c083f44.jpg)
![20210921_161032](https://user-images.githubusercontent.com/78416937/134198183-cd4c445c-df26-4317-bb32-8113725b22e3.jpg)

**Comment by MIKE-DAILEY** [comment]
Not able to replicate on X20S with 1.0.11. FCC.

**Comment by loetefix** [comment]
Maybe a hardware problem. Is it still with an other firmware?

**Comment by Franjose1984** [comment]
I have tried another version of the firmware and the problem persists ... I don't know what I could check.

**Comment by loetefix** [comment]
It seems this is a hardware problem.

**Comment by Franjose1984** [comment]
I have decided to look inside the TX and I have checked all the wiring, everything is correct, I have checked with a multimeter if there is any mixture between these two pots and everything is correct, so I have been able to inform myself I think it may be an error in the TX board readings between these two pots generating a mix between them. It makes me think that it may be more of a firmware error than a hardware one.
At the moment the solution I have taken is to modify the curve of the channel of the left pot.
![20211004_122509](https://user-images.githubusercontent.com/78416937/135843931-364b4774-1716-482f-9f68-5234d200be5e.jpg)
